agent/config/config_test.go

```diff
@@ -494,6 +498,62 @@ func TestValidFormatParseEnvVariableDuration(t *testing.T) {
 	assert.Equal(t, 1*time.Second, duration, "Unexpected value parsed in parseEnvVariableDuration.")
 }
 
+func TestValidFormatParseDynamicHostPortRange(t *testing.T) {
```

Comment:
nit: I see that the test has multiple cases now (both success and failure), so `TestParseDynamicHostPortRange` seems more suited.